packages/api-types/src/vessel.ts

```diff
@@ -17,7 +23,7 @@ export interface Vessel {
   owner?: string
   operator?: string
   builtYear?: string
-  authorizations?: string[]
+  authorizations?: Authorization[]
```

Comment:
You'll have to publish a new version of this package

Reply:
yes